doc/dev/rbd-export.rst

```diff
@@ -65,6 +65,20 @@ Image Stripe count
 - le64: length of appending data (8)
 - le64: image striping count
 
+ImageMeta Key
```

Comment:
Image-meta should be combined into a single tag -- so perhaps the 'M' tag (for image-**m**eta), le64 for tag length as usual, then the string key followed by the string value.

Reply:
what you suggest is that we should combined all key-value paires into one tag?